templates/books-index.html

```diff
@@ -0,0 +1,30 @@
+<!-- header -->
+<div class="jumbotron">
+  <div class="container">
+    <div class="col-md-6">
+      <h1 class="yay">YAY BOOKS!</h1>
```

Comment:
Might want to use a more general class name in case you want to change the text in the future.  Ex: "jumbotron-text"